### ISSUE TITLE
[BUG]: <short description of the issue>

### ISSUE THREAD

**Issue by medsabbar** [serengil/deepface#1476]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [x] I have updated to the latest version of the packages.
- [x] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

v0.0.93

### Python version

3.9

### Operating System

Ubuntu 22

### Dependencies

absl-py==2.2.2
astunparse==1.6.3
beautifulsoup4==4.13.4
blinker==1.9.0
certifi==2025.4.26
charset-normalizer==3.4.2
click==8.1.8
deepface==0.0.93
filelock==3.18.0
fire==0.7.0
Flask==3.1.1
flask-cors==5.0.1
flatbuffers==25.2.10
fsspec==2025.3.2
gast==0.6.0
gdown==5.2.0
google-pasta==0.2.0
grpcio==1.71.0
gunicorn==23.0.0
h5py==3.13.0
idna==3.10
importlib_metadata==8.7.0
itsdangerous==2.2.0
Jinja2==3.1.6
keras==3.9.2
libclang==18.1.1
Markdown==3.8
markdown-it-py==3.0.0
MarkupSafe==3.0.2
mdurl==0.1.2
ml_dtypes==0.5.1
mpmath==1.3.0
mtcnn==0.1.1
namex==0.0.9
networkx==3.2.1
numpy==2.0.2
nvidia-cublas-cu12==12.6.4.1
nvidia-cuda-cupti-cu12==12.6.80
nvidia-cuda-nvrtc-cu12==12.6.77
nvidia-cuda-runtime-cu12==12.6.77
nvidia-cudnn-cu12==9.5.1.17
nvidia-cufft-cu12==11.3.0.4
nvidia-cufile-cu12==1.11.1.6
nvidia-curand-cu12==10.3.7.77
nvidia-cusolver-cu12==11.7.1.2
nvidia-cusparse-cu12==12.5.4.2
nvidia-cusparselt-cu12==0.6.3
nvidia-nccl-cu12==2.26.2
nvidia-nvjitlink-cu12==12.6.85
nvidia-nvtx-cu12==12.6.77
opencv-python==4.11.0.86
opt_einsum==3.4.0
optree==0.15.0
packaging==25.0
pandas==2.2.3
pillow==11.2.1
protobuf==5.29.4
Pygments==2.19.1
PySocks==1.7.1
python-dateutil==2.9.0.post0
pytz==2025.2
requests==2.32.3
retina-face==0.0.17
rich==14.0.0
six==1.17.0
soupsieve==2.7
sympy==1.14.0
tensorboard==2.19.0
tensorboard-data-server==0.7.2
tensorflow==2.19.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==3.1.0
tf_keras==2.19.0
torch==2.7.0
tqdm==4.67.1
triton==3.3.0
typing_extensions==4.13.2
tzdata==2025.2
urllib3==2.4.0
Werkzeug==3.1.3
wrapt==1.17.2
zipp==3.21.0

### Reproducible example

```Python
this photo and others return face is real
https://odoobackup.fra1.digitaloceanspaces.com/download%20(1).jpeg
```

**Comment by serengil** [comment]
individual misclassifications don't make the model wrong

**Comment by medsabbar** [comment]
Thanks for your reply
